### PR TITLE
List some differences between callback actors and other actors

### PR DESCRIPTION
**Pull request by kevinmaes**

### CODE REVIEW

File: versioned_docs/version-5/actors.mdx [hunk 1]
```diff
@@ -352,6 +352,13 @@ const callbackLogic = fromCallback((sendBack, receive) => {
 });
 ```
 
+Callback actors are a bit different from other actors in that they do not do the following:
+
+- Do not work with onDone or onError
+- Do not produce a snapshot using .getSnapshot()
```
Comment (laurakalbag) [inline versioned_docs/version-5/actors.mdx:358]:
```suggestion
- Do not produce a snapshot using `.getSnapshot()`
```

File: versioned_docs/version-5/actors.mdx [hunk 2]
```diff
@@ -352,6 +352,13 @@ const callbackLogic = fromCallback((sendBack, receive) => {
 });
 ```
 
+Callback actors are a bit different from other actors in that they do not do the following:
+
+- Do not work with onDone or onError
```
Comment (laurakalbag) [inline versioned_docs/version-5/actors.mdx:357]:
```suggestion
- Do not work with `onDone` or `onError`
```

File: versioned_docs/version-5/actors.mdx [hunk 3]
```diff
@@ -352,6 +352,13 @@ const callbackLogic = fromCallback((sendBack, receive) => {
...
+- Do not produce a snapshot using .getSnapshot()
+- Do not emit values when used with .subscribe()
```
Comment (laurakalbag) [inline versioned_docs/version-5/actors.mdx:359]:
```suggestion
- Do not emit values when used with `.subscribe()`
```

File: versioned_docs/version-5/actors.mdx [hunk 4]
```diff
@@ -352,6 +352,13 @@ const callbackLogic = fromCallback((sendBack, receive) => {
...
+- Can not be stopped with .stop()
```
Comment (laurakalbag) [inline versioned_docs/version-5/actors.mdx:360]:
```suggestion
- Can not be stopped with `.stop()`
```

**Review by laurakalbag — APPROVED** [review]
Added a few suggestions for formatting nits. This is great, thank you!

**Comment by kevinmaes** [comment]
The backtick styles for code bits makes total sense. Thanks for the review @laurakalbag!

**Pull request merged**